app/javascript/packs/application.js

```diff
@@ -3,10 +3,11 @@
 // a relevant structure within app/javascript and only use these pack files to reference
 // that code so it'll be compiled.
 
-import { Turbo, cable } from '@hotwired/turbo-rails';
-import * as ActiveStorage from '@rails/activestorage';
-import 'channels';
-import 'controllers';
+import { Turbo, cable } from "@hotwired/turbo-rails";
```

Comment:
We can probably delete this import on line 6, since it's essentially duplicated on line 7. Line 7 seems to be the preferred import approach now.